src/main/java/at/hannibal2/skyhanni/config/features/inventory/experimentationtable/ExperimentsProfitTrackerConfig.java

```diff
@@ -31,6 +31,12 @@ public class ExperimentsProfitTrackerConfig {
     @ConfigEditorSlider(minValue = 5, maxValue = 60, minStep = 1)
     public int timeDisplayed = 30;
 
+    @Expose
+    @ConfigOption(name = "Ironman Profit Calculations", desc = "Removes the Cost of Experience Bottles from Profit Calculations. (Requires Restart)")
+    @ConfigEditorBoolean
+    public boolean isIronman = false;
+    // Unable to make the Overlay update automatically, needs fixing!! Therefore the "Requires Restart" note
```

Comment:
you can auto update the data:
1. change the config option from boolean to `Property<Boolean> name = Propety.of(default value)`
2. create a `ConfigLoadEvent` and a `onToggle`, `afterChange` or `whenChanged` function to listen to the config value change
3. when the value changes, set `TrackerManager.dirty` to `true`

search for "property.of" in the config, and see how those values are getting used for many examples.